docs/first-order/modality_alignment.ipynb

```diff
@@ -0,0 +1,600 @@
+{
```

Comment:
<p>hertz is a unit of measure not a description of the measure. Just replace to "they must be aligned to the same time points." Same for the last sentence</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AllenInstitute/openscope_databook/pull/223/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>